CHANGELOG.md

```diff
@@ -5,42 +5,45 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased][unreleased]
```

Comment:
i don't mind changing it, but they are both just different, equally valid ways to generate markdown [links](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links) and render identically.